--- qjoypad-4.1.0/src/qjoypad.pro
+++ qjoypad-4.1.0/src/qjoypad.pro
@@ -15,8 +15,6 @@
 icons.conf_path = $$PREFIX/share/pixmaps/qjoypad
 icons.extra = cp ../icons/* $${icons.path}; cd $${icons.path}; ln -sf gamepad4-24x24.png icon24.png; ln -sf gamepad3-64x64.png icon64.png; chmod -R a+r $${icons.path}
 
-doc.path = $$INSTALL_PREFIX/share/doc/qjoypad4
-doc.extra = cp ../README.txt ../LICENSE.txt $${doc.path}
 target.path = $$INSTALL_PREFIX/bin
 
 
@@ -76,4 +74,4 @@
 
 #####      Install      #####
 
-INSTALLS += target icons doc
+INSTALLS += target icons
